src/Delivery/SimpleResponseParser.php

```diff
@@ -55,7 +62,7 @@ protected function resolveEntry(DynamicEntry $entry): array
         }, $fields);
 
         return array_filter($return, function ($value) {
-            return (is_string($value) && $value !== '') || is_bool($value) || !is_scalar($value) || is_numeric($value);
+            return !is_null($value) ||(is_string($value) && $value !== '') || is_bool($value) || !is_scalar($value) || is_numeric($value);
```

Comment:
Can you explain what do you here test?

String and bool is ok, but scalar is not ok? But string and bool are scalar values ...

Reply:
Change is not required anymore